include/fuse.h

```diff
@@ -984,6 +987,10 @@ int fuse_loop(struct fuse *f);
  */
 void fuse_exit(struct fuse *f);
 
+#if FUSE_USE_VERSION < 32
```

Comment:
Same question here - why is this needed?

Reply:
the inline diff is a bit weird, but I've moved the comment line just before the function definition.

Before, the comment block was decorating 
int fuse_loop_mt_31(struct fuse *f, int clone_fd);

and there was a warning that `clone_fd` isn't described, and `config` isn't defined.